.github/workflows/tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     if: github.event_name == 'pull_request' # pre-commit-ci/lite-action only runs here
     strategy:
       matrix:
-        python-version: [3.11, 3.13] # Our min and max supported Python versions
```

Comment:
Let's not disable 3.13 entirely in CI due to `pylint`, let's just do `3.13.0` instead